microsoft-edge/devtools-guide-chromium/3D-view.md

```diff
@@ -0,0 +1,77 @@
+---
+description: All about 3D View and how to use it.
+title: 3D View
+author: erdraud
+ms.author: erdraud
+ms.date: 11/27/2019
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+
+# 3D View
+
+Use the **3D View** to debug your web application by navigating through the Document Object Model (DOM) or the z-index stacking context. With it you can: 
```

Comment:
I think it would be beneficial to link to the MDN pages for the DOM and z-index as I think those pages provide some valuable info to readers. Here's the link for the DOM: https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model and here's the link for z-index: https://developer.mozilla.org/en-US/docs/Web/CSS/z-index